OpenTaiko/src/Songs/CDTX.cs

```diff
@@ -5123,7 +5123,7 @@ string[] SplitComma(string input) {
 				chip.nコース = this.n現在のコース;
 
 				// チップを配置。
-				eScrollMode = EScrollMode.BMSCROLL;
+				eScrollMode = EScrollMode.BmScroll;
```

Comment:
Probably a name conversion issue, but they are capitalized as respectively "BMScroll" and "HBScroll" in TaikoJiro. "BM" is for "BMS" (Be-Music Source) and "HB" is for "Hi-Speed with BMS".